### PR TITLE
Update Che Theia CSS styling to respect the VS Code theming approach

### PR DESCRIPTION
**Pull request by azatsarynnyy**
Signed-off-by: Artem Zatsarynnyi <azatsary@redhat.com>

### What does this PR do?
Updates Che Theia CSS styling to respect the VS Code theming approach.

Below is an example of Che Theia Plugins view with the different VS Code themes applied.

themes

### What issues does this PR fix or reference?
closes https://github.com/eclipse/che/issues/16079

### CODE REVIEW

File: extensions/eclipse-che-theia-plugin-ext/src/browser/style/che-plugins-caching.css [hunk 1]
```diff
@@ -39,5 +39,5 @@
 }
 
 .che-plugins-caching-progress-status b {
-    color: var(--theia-success-color2);
+    color: var(--theia-successBackground);
```
Comment (vitaliy-guliy) [inline extensions/eclipse-che-theia-plugin-ext/src/browser/style/che-plugins-caching.css]: color property sets a text color. Could you pick another variable, not  --theia-successBackground ?

File: extensions/eclipse-che-theia-plugin-ext/src/browser/style/che-plugins.css [hunk 2]
```diff
@@ -80,23 +80,23 @@
...
     border: none;
-    background: var(--theia-menu-color1);
+    background: var(--theia-menubar-selectionBackground);
     box-shadow: 0px 0px 6px rgba(0, 0, 0, 0.2);
 }
 
 .che-plugin-control-panel .menu-button-disabled {
     outline: none;
-    color: var(--theia-ui-font-color3);
+    color: var(--theia-badge-background);
```
Comment (vitaliy-guliy) [inline extensions/eclipse-che-theia-plugin-ext/src/browser/style/che-plugins.css]: color property sets a text color. Could you pick another variable, not --theia-successBackground ?

File: extensions/eclipse-che-theia-plugin-ext/src/browser/style/che-plugins-caching.css (hunk 1 above)
Comment (azatsarynnyy) [reply extensions/eclipse-che-theia-plugin-ext/src/browser/style/che-plugins-caching.css]: updated to `--theia-badge-foreground` which is best suited for that

File: extensions/eclipse-che-theia-plugin-ext/src/browser/style/che-plugins.css (hunk 2 above)
Comment (azatsarynnyy) [reply extensions/eclipse-che-theia-plugin-ext/src/browser/style/che-plugins.css]: Sure, just choose any that works best for you )
https://code.visualstudio.com/api/references/theme-color

**Comment by azatsarynnyy** [comment]
@benoitf VS Code themes were added just for testing the PR while it was in draft mode.
Do you think we need to include VS Code built-in themes in Che Theia by default?

**Comment by benoitf** [comment]
@azatsarynnyy we could propose a vote :-)

I don't know, but for example if a user install VS Code he gets built-in themes and extensions so maybe default themes are expected as well ?

**Comment by azatsarynnyy** [comment]
I'll propose it in a separate PR

**Comment by azatsarynnyy** [comment]
crw-ci-test

**Comment by azatsarynnyy** [comment]
crw-ci-test

**Comment by azatsarynnyy** [comment]
ATM Happy Path tests are failing due to installation issue https://www.eclipse.org/lists/che-dev/msg03678.html
I'm merging it since the PR just updates some colors in styles and previous run of Happy Path was successful, see https://github.com/eclipse/che-theia/pull/656#issuecomment-596804251

**Pull request merged**